test-utils/state-viewer/src/main.rs

```diff
@@ -210,6 +213,210 @@ fn replay_chain(
     }
 }
 
+#[derive(Clone, Debug)]
+struct ApplyInfo {
+    shard_id: ShardId,
+    block_height: BlockHeight,
+    block_hash: CryptoHash,
+    chunk_extra: ChunkExtra,
+    existing_chunk_extra: Option<ChunkExtra>,
+    outcomes: Vec<ExecutionOutcomeWithId>,
+}
```

Comment:
Please add a comment on what the struct is for.

Reply:
Removed